schema.sql

```diff
@@ -49,3 +49,22 @@ ALTER TABLE animals
 ADD COLUMN owner_id INTEGER REFERENCES owners(id);
 
 
```

Comment:
`README.md` file:

- [ ] I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same content in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**
![Screenshot from 2023-09-27 17-07-06](https://github.com/Kaiserabbas/vet_clinic/assets/83042742/c8ae5348-6401-45cf-81e1-e732a05bdffe)